### PR TITLE
Add script organization slides

### PR DESCRIPTION
**Pull request by jashapiro**
Related to #25 and #27, this adds links to the slides for script and notebook content organization.

Review should be focused on the slides themselves, which can be found at 
https://docs.google.com/presentation/d/1AJr6uQhwLnZfis1wNc_e2XY4XSMEuVscIfAsVgnM5Bk/edit?usp=sharing

These slides were a bit more of a struggle than I expected, as I had a _lot_ of things I wanted to say. So organization changed a lot as things were written... see if you think it makes sense as it is or if you have ideas for changing organization. 

Also if there are things I missed that you think are imorportant, or things that you think I over or under-emphasize, please leave those comments!

### CODE REVIEW

File: instructor_notes/06_script-content-and-stitching.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+
+* In RStudio, have trainees open `analyses/mutation_counts/01_count-gene-mutations.R`
+* Briefly walk through the components of the R script, including the following:
+  * header comments
+  * package imports
+  * option setting & error checking
+  * constants (note that in this case these are set after options)
+
+* During the above (notably option setting) demonstrate usage of the r script
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:30]:
```suggestion
* During the above (notably option setting) demonstrate usage of the R script
```

File: workshop_materials/slides/README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@ Links to the Google Slides we present as part of the workshop can be found below
 
 1. [Project Organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing)
 2. [Introduction to UNIX](https://docs.google.com/presentation/d/1WPXkItJZEUXMY20cLrdMXHiBC2PyunR14RUSDg4nfIc/edit?usp=sharing)
-3. [Introduction to Git; forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing)
+3. [Introduction to Git; Forking and Cloning a Repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing)
 4. [Shell Scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing)
-5. [The Git Stage/Commit/Push workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)
-6. Organizing Scripts and Notebooks
+5. [The Git Stage/Commit/Push Workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)
```
[the remark this answers is not in the document]
Comment (jashapiro) [reply workshop_materials/slides/README.md:9]: I thought it was probably fine?

File: instructor_notes/06_script-content-and-stitching.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+  ```sh
+  cd analyses/mutation_counts # if needed
+
+  Rscript 01_count-gene-mutations.R -h
+  ```
+  * Show the command running with correct options
+
+  ```sh
+  Rscript 01_count-gene-mutations.R --maf ../../data/processed/maf_files/LGAT.maf.tsv.gz -o LGAT_gene-mutations.tsv
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:42]: Question about this - Are we intentionally showing them the two different syntaxes (synti? syntaxi?!?!) for providing the arguments? Would be worth pointing out either way that you can mix and match.

**Comment by jashapiro** [comment]
I added prepended notes to the stitching activity here (and renamed the file to be more inclusive). 

There have also been slide updates in response to comments there, so you might want to have a look at those again if you have not already.

File: instructor_notes/06_script-content-and-stitching.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  ```
+
+  * Possibly: include a typo in the MAF file, and show the error message!
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:45]: Always love a good intentional mistake.

File: instructor_notes/06_script-content-and-stitching.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+* Have the trainees open the rendered notebook `analyses/mutation_counts/01_count-gene-mutations.nb.html` in the browser
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:47]:
```suggestion
* Have the trainees open the rendered notebook `analyses/mutation_counts/02_mutation-count-plots.nb.html` in the browser.
```

File: instructor_notes/06_script-content-and-stitching.md [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+  * Point out the lovely rendering that we get for free!
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:48]: Worth emphasizing too that an HTML file **is simply a file.** It's just a file that opens in the browser apps but it's not always the internet. HTML files can be shared and opened without regard to internet connection!

File: instructor_notes/06_script-content-and-stitching.md [hunk 7]
```diff
@@ -1,23 +1,63 @@
...
+* Examine the structure of provided shell scripts
+* Run R scripts with arguments from the command line, including the use of `--help`
+* Render Rmarkdown documents/notebooks on the command line
+* Understand how shell scripts can be used to automate multi-step analyses
+* Recognize some limitations of shell scripting to automate analyses
 
+## Activity Part 1: Exploring & running the R script & notebook
 
-## Activity
+At this point we will have completed a slide presentation that describes the R script contents.
```
Comment (sjspielman) [inline instructor_notes/06_script-content-and-stitching.md:19]:
```suggestion
At this point we will have completed a slide presentation that describes the R script and notebook contents.
```

**Review by sjspielman — APPROVED** [review]
Ready to go, just one quick change since the notebook is also discussed in the slides. Don't need to see again.

**Pull request merged**